backend/game_engine/events.py

```diff
@@ -105,36 +105,42 @@ def _delete_message(self, receipt_handle: Text) -> None:
                                     ReceiptHandle=receipt_handle)
 
     def get(self) -> SMSEvent:
+        game_id = ""
```

Comment:
in cases where we'd like to log messages from objects which do not have a game_id explicitly set in the constructor, we may need to modify the constructor to accept a game_id and ensure that it's populated.